build/Publish.targets

```diff
@@ -174,6 +175,10 @@
       <PackageToPublish Include="%(_SymbolsPackageArtifactInfo.ArtifactPath)" Category="symbols" IsSymbolsPackage="true" />
     </ItemGroup>
 
+    <ItemGroup>
+      <RedistPackageToPublish Include="$(DependencyAssetsDir)(SharedFrameworkRedistPackageId)*" />
```

Comment:
Well don't I have an egg on my face... This isn't working because there's no '$' before `(SharedFrameworkRedistPackageId)` 🤦‍♂